test/javascripts/helpers/create-store.js.es6

```diff
@@ -16,6 +17,11 @@ export default function() {
           }
           return this._restAdapter;
         }
+        if (type === "adapter:topicList") {
```

Comment:
Why is this necessary? Shouldn't the topic list adapter automatically be returned by name?

Reply:
In test environment, `store` seems to be stubbed by this `create-store.js.es6` helper. It expects an exact match when `lookup`, and would otherwise [fallback to `adapter:rest`](https://github.com/discourse/discourse/blob/f47ae17d6dc92addd473e02470c78f03fd3eb3ed/app/assets/javascripts/discourse/models/store.js.es6#L258).